openwisp_controller/config/who_is/test_who_is.py

```diff
@@ -0,0 +1,355 @@
+import importlib
```

Comment:
The naming of this file is not consistent with the conventions used in openwisp, we should rename it to `test_whois.py`.